### PR TITLE
Added user profile view and edit feature

### PR DESCRIPTION
**Pull request by adrean-hipe**
OVERVIEW

- Added ProfileController for user profile
- Used request in updating user information
- Used service in updating the profile
- Added seeder for user information that matches the seeder in user table
- Added initial UI for viewing user profile
- Added initial UI for editing user profile
- Added routing for the relevant profile functions (index, edit, update)

NOTES

- Added other areas as fillable in userInformation table to be able to update details
- To be added profile image upload feature
- Initial design for the UI, will be improved

image
image
image

### CODE REVIEW

**Comment by mikco-hipe** [comment]
LGTM, testing lang po hehe

File: laravel-app/public/assets/css/app.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+	/* overflow-x: hidden; */
+}
+
+#app{
+	min-height: 100vh;
+}
+
+#profile-header{
+	min-height: 30vh;
+}
```
Comment (bpocrafael) [inline laravel-app/public/assets/css/app.css:11]: add new line

File: laravel-app/routes/web.php [hunk 2]
```diff
@@ -27,6 +28,9 @@
     Route::post('/logout', [HomeController::class, 'logout'])->name('logout');
     Route::get('/post', [PostController::class, 'index'])->name('post.index');
     Route::post('/post', [PostController::class, 'store'])->name('post.store');
+    Route::get('/profile', [ProfileController::class, 'index'])->name('profile.index');
```
Comment (bpocrafael) [inline laravel-app/routes/web.php:31]: can you make a resource group of this??

File: laravel-app/resources/views/profile/profile-form.blade.php [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+						<button type="submit" class="btn btn-success">
+							{{ __('Update') }}
+						</button>
+					</div>
+				</div>
+
+				<div class="row p-2 mb-0 mt-3">
+				</div>
+	
+			</form>
```
Comment (bpocrafael) [inline laravel-app/resources/views/profile/profile-form.blade.php:114]: add new line

File: laravel-app/public/assets/css/app.css (hunk 1 above)
Comment (hipe01mark) [inline laravel-app/public/assets/css/app.css:11]: add new line

File: laravel-app/resources/views/profile/profile-form.blade.php (hunk 3 above)
Comment (hipe01mark) [inline laravel-app/resources/views/profile/profile-form.blade.php:114]: add new line

**Review by hipe01mark — DISMISSED** [review]
LGTM 🙇

File: laravel-app/app/Http/Controllers/ProfileController.php [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        /** @var User $user */
+        $user = auth()->user();
+        
+        try {
+            $this->userService->updateProfile($user, $request->all());
+
+            return redirect()->route('profile.edit')->with('success', 'Profile updated successfully');
+        } catch (QueryException $e) {
+            if ($e->errorInfo[1] === 1062) {
+                return redirect()->back()->withErrors(['email' => 'The email address is already in use. Please choose a different email.']);
```
Comment (bpocrafael) [inline laravel-app/app/Http/Controllers/ProfileController.php:42]: too many lines in one statement you can breakdown that has a long statement.
nit,

File: laravel-app/public/assets/css/app.css (hunk 1 above)
Comment (adrean-hipe) [reply laravel-app/public/assets/css/app.css]: added new line sir. Thank you ❤️

File: laravel-app/resources/views/profile/profile-form.blade.php (hunk 3 above)
Comment (adrean-hipe) [reply laravel-app/resources/views/profile/profile-form.blade.php]: added new line sir and fixed indention. Thank you ❤️

File: laravel-app/public/assets/css/app.css (hunk 1 above)
Comment (adrean-hipe) [reply laravel-app/public/assets/css/app.css]: added new line sir. Thank you ❤️

File: laravel-app/routes/web.php (hunk 2 above)
Comment (adrean-hipe) [reply laravel-app/routes/web.php]: Changed to resource group for profile routing sir. Thank you ❤️

File: laravel-app/resources/views/profile/profile-form.blade.php (hunk 3 above)
Comment (adrean-hipe) [reply laravel-app/resources/views/profile/profile-form.blade.php]: added new line sir and fixed indention. Thank you ❤️

File: laravel-app/app/Http/Controllers/ProfileController.php (hunk 4 above)
Comment (adrean-hipe) [reply laravel-app/app/Http/Controllers/ProfileController.php]: Added error message to variable to break down statement, sir. Thank you ❤️

**Review by bpocrafael — APPROVED** [review]

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Pull request merged**